### DISCUSSION TITLE
Long events notification delay

### DISCUSSION THREAD

**Discussion by harpoo123** [blakeblackshear/frigate#2655] [Ask A Question]
Hi,

I have noticed that when Frigate detects an object (a person for instance) it will not create the event until that person goes outside the camera shot. This causes a delay on the notification.

I was wondering whether there is a setting you can use to force Frigate to create the event after a number of seconds from the detection.

If any of you has come across the same issue and have found a way around it would be great if you shared your approach here.

Many thanks

**Comment by blakeblackshear** [comment]
This shouldn't create a delay in the notification unless you are waiting for the end event type.
